lib/Conversion/TorchToMhlo/Reduction.cpp

```diff
@@ -336,7 +343,8 @@ LogicalResult ConvertAtenReductionOp<AtenSumOp>::matchAndRewrite(
   Value input = adaptor.self();
   auto inputTy = input.getType().dyn_cast<RankedTensorType>();
   if (!inputTy) {
-    return rewriter.notifyMatchFailure(op, "only Tensor types supported in MHLO");
+    return rewriter.notifyMatchFailure(op,
```

Comment:
can you submit this reformatting as a separate PR? It is hard to see the actual change here.

Reply:
Sorry for that. I will revert the modification of reformatting.